### PR TITLE
Responsive navbar updated

### PR DESCRIPTION
**Pull request by Priyaraj17**
Added responsive behavior for smaller screens.

## Summary

The UI wasn't working for smaller screens especially the navbar. I updated the navbar and added responsiveness for smaller screen sizes such as 768px and 380px.  I replaced some of the classes for navbar.  From 768px the navbar disappears and a hamburger is present.

## Motivation

The user interface was not very attractive for smaller screens. I created this issue when I saw the bug and created a PR for it.

## Testing

I used chrome dev tools for testing and these are the images:

image

image

### CODE REVIEW

File: js/templatemo-script.js [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
+    $("#rotate").rotaterator({
+      fadeSpeed: 1000,
+      pauseSpeed: 300,
+    });
+  };
 
-    // Update year in copyright text
-    document.querySelector('.tm-current-year').textContent = new Date().getFullYear();
+  // Update year in copyright text
+  document.querySelector(
```
Comment (birm) [inline js/templatemo-script.js:60]: That's certainly better than us forgetting to update it each time :)

**Review by birm — APPROVED** [review]
Looks good to me, thank you!

**Comment by Priyaraj17** [comment]
> Looks good to me, thank you!

Thank you.
I also noticed a few things that can be improved here. I was thinking of using rems instead of pixels that will ensure things will be fine when a user zooms in or out. I believe its a good convention to use them.

**Comment by Priyaraj17** [comment]
> Looks good to me, thank you!

Hey, is this PR okay to be merged?

File: css/templatemo-style.css [hunk 2]
```diff
@@ -7,11 +7,11 @@ http://www.templatemo.com/tm-517-timeless
 */
```
Comment (sohamsshah) [inline css/templatemo-style.css:7]: Just curious that is this file needed now in the project? I suppose this file was the styling of the older website. If not needed, we can remove this in this PR itself (or a new PR is also okay). I suspect a lot of changes in this file, probably due to code editor.

**Review by sohamsshah — APPROVED** [review]
Great work man! Loved it :)
Thanks for working on it @Priyaraj17

**Comment by Priyaraj17** [comment]
@sohamsshah happy to help 😊.
